CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 2.5.0
+### Visual changes
+* Move "sync axis" and "show only main axis" settings to "Axis" card
+* Add font setttings for measure units and axis
+* Add an 'auto width' toggle for category labels that, when enabled, adjusts the label width to display all categories without trimming
```

Comment:
Split this sentence to make it more laconic: 
`Add "Auto width" toggle for category labels. When enabled, this feature dynamically adjusts the label width to ensure all category names are fully visible without truncation.`